service/policy/db/queries/attributes.sql

```diff
@@ -3,6 +3,11 @@
 ----------------------------------------------------------------
 
 -- name: listAttributesDetail :many
+WITH params AS (
+    SELECT
+        NULLIF(@namespace_id, '')::uuid as namespace_id,
```

Comment:
Is there a reason we cannot unify on `sqlc.narg` or `NULLIF` exclusively in all of these queries rather than using both? It looks like `sqlc.narg` is a little cleaner.

Reply:
`sqlc.narg` and `NULLIF` serve different purposes, and I didn't want to change app code (to prove these SQL changes didn't break anything). However, the app code changes were simple. Removed all `NULLIF` invocations from all SQL queries.